libraries/ms-common/inc/ms_test_helper_datagram.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+/* Test helpers for datagrams
+ */
```

Comment:
We don't usually use C-style /* comments */ - could you replace it with a `//` comment and then add a `// Requires ... to be initialized` comment?